frontend/src/pages/Profile/loader.ts

```diff
@@ -0,0 +1,15 @@
+import { Params } from "react-router-dom";
+import axios from "axios";
+
+const loader = async ({ params }: { params: Params }) => {
+  // TODO: Check the JWT, get the user from it, and get his data
```

Comment:
jak TODO to i nr taska w którym ten TODO ma być zrobiony. Łatwiej się potem taska robi bo wyszukujesz wszystkie miejsca po ctrl+f